public/usage-examples/physics/translation_matrix/translation_matrix-simple.cpp

```diff
@@ -0,0 +1,25 @@
+#include "splashkit.h"
+
+int main()
+{
+    // Define the translation values
+    double translation_x = 200;
+    double translation_y = 100;
+
+    // Create a translation matrix using the translation values
+    matrix_2d my_matrix_1 = translation_matrix(translation_x, translation_y);
```

Comment:
this matrix variable has a different name in cpp than it does in all other files. I would recommend updating the other files as their variable names are the same as the function called, which can potentially cause confusion 
![image](https://github.com/user-attachments/assets/90522dd6-7ed1-4ce9-bf11-8fb7e143e0ab)